jetstream/1-click-set-to-default-existing-users.toml

```diff
@@ -0,0 +1,22 @@
+[experiment]
+
+[experiment.exposure_signal]
+description = "filters to clients with experiment's spotlight message impression"
+name = "spotlight_exposure"
+friendly_name = "Spotlight Exposure"
+select_expression = """(
+    COALESCE(mozfun.map.get_key(ping_info.experiments, '1-click-set-to-default-existing-users') IS NOT NULL
+             AND metrics.text2.messaging_system_message_id = 'FOX_DOODLE_SET_DEFAULT', FALSE)
+)"""
+data_source = "spotlight_impressions"
+window_start = 0
+window_end = 7
+
+
+[data_sources]
+[data_sources.spotlight_impressions]
+from_expression = '''(SELECT *, metrics.uuid.messaging_system_client_id AS client_id,
+DATE(submission_timestamp) AS submission_date FROM `moz-fx-data-shared-prod.firefox_desktop.messaging_system` WHERE metrics.string.messaging_system_ping_type = 'spotlight')'''
+experiments_column_type = "native"
```

Comment:
I think this should be `glean` not `native`. Although, I also think that the CI should've errored here

Reply:
maybe it didn't because of the `AS client_id` in `metrics.uuid.messaging_system_client_id AS client_id`?

Reply:
i did run jetstream validate_config and it did throw an error when i didn't have that

Reply:
These fields don't appear in the [dry run query](https://app.circleci.com/pipelines/github/mozilla/metric-hub/2327/workflows/23dc2c79-fe4b-4125-b8e3-206d63a0a3d5/jobs/4844) that we use to test the configuration

Reply:
sounds good, i'll make the change